js/src/components/tree-select-control/index.js

```diff
@@ -87,6 +88,7 @@ import { ARROW_DOWN, ARROW_UP, ENTER, ESCAPE, ROOT_VALUE } from './constants';
  * @param {Option[]} [props.options] Options to show in the component
  * @param {string[]} [props.value] Selected values
  * @param {number} [props.maxVisibleTags] The maximum number of tags to show. Undefined, 0 or less than 0 evaluates to "Show All".
+ * @param {boolean} [props.allowsSelectParents=false] If true, when selecting a parent it also adds its key in the onChange callback.
```

Comment:
Looks like it's supposed to be *its value*.

```suggestion
 * @param {boolean} [props.allowsSelectParents=false] If true, when selecting a parent it also adds its value in the onChange callback.
```